frontend/src/components/DropDown/DropDown.tsx

```diff
@@ -15,15 +15,13 @@ function DropDown({ element }: { element: JSX.Element[] }): JSX.Element {
 	);
 
 	return (
-		<Dropdown menu={{ items }} className="Dropdown-container">
+		<Dropdown menu={{ items }}>
 			<Button
 				type="link"
 				className={!isDarkMode ? 'Dropdown-button--dark' : 'Dropdown-button'}
 				onClick={(e): void => e.preventDefault()}
 			>
-				<Space>
-					<EllipsisOutlined className="Dropdown-icon" />
-				</Space>
+				<EllipsisOutlined className="Dropdown-icon" />
```

Comment:
Same old comment - className should be camelCased or lower-case / lowercase - hyphenated.

Reply:
Done.